presto-yarn-package/src/main/slider/package/templates/resource_groups.json.j2

```diff
@@ -0,0 +1,28 @@
+{
+  "rootGroups": [
+    {
+      "name": "global",
+      "softMemoryLimit": "80%",
+      "hardConcurrencyLimit": 100,
+      "maxQueued": 1000,
+      "schedulingPolicy": "weighted",
+      "jmxExport": true,
+      "subGroups": [
+        {
+          "name": "adhoc_${USER}",
+          "softMemoryLimit": "10%",
+          "hardConcurrencyLimit": 2,
```

Comment:
how use can override these defaults? IIRC it is not possible. Do you think it is achievable to make it possible to for user to control these groups properties?

Reply:
I am not sure if it is possible. I have not seen any documentation for doing this. Meanwhile even query queues didnt support user config.